### DISCUSSION TITLE
Need help for installing the latest version of DP-GEN off-line. Thank you!

### DISCUSSION THREAD

**Discussion by wxwth** [deepmodeling/dpgen#1452] [Q&A]
Dear DP-GEN developers,
    I have the same issue as post #790 , and have to intall DP-GEN off-line. However, I observe that there is no updated .sh file for DP-GEN beyond version 0.10.6. I am wondering if there is any method to install the latest version of DP-GEN off-line. Thanks in advance!
Sincerely,
Xu Wang

**Comment by njzjz** [comment]
We used [conda constructor](https://github.com/conda/constructor) to pack conda environments. You can pack any packages you want by using it.

**Reply by biglinn** [reply]
Mr.Zeng, could you please give me some help? I have a problem about dpgen run.#1463
